joint_trajectory_controller/include/joint_trajectory_controller/hardware_interface_adapter.h

```diff
@@ -213,7 +216,7 @@ class HardwareInterfaceAdapter<hardware_interface::VelocityJointInterface, State
     // Update PIDs
     for (unsigned int i = 0; i < n_joints; ++i)
     {
-      const double command = pids_[i]->computeCommand(state_error.position[i], state_error.velocity[i], period);
+      const double command = (desired_state.velocity[i] * use_velocity_ff_) + pids_[i]->computeCommand(state_error.position[i], state_error.velocity[i], period);
```

Comment:
Is it the unconventional `double * bool` multiplication which you don't like or is it restricting the range of values to effectively `0` and `1`? If its the multiplication I can always reformulate in a conditional statement.

I ask because I can't think of any reasonable case where feedforward _gains_ other than `0` or `1` make sense. Can you?